_data/simple-icons.json

```diff
@@ -7093,6 +7093,27 @@
             "hex": "CC2927",
             "source": "https://de.wikipedia.org/wiki/Datei:Microsoft_SQL_Server_Logo.svg"
         },
+        {
+            "title": "Microsoft SQL Server Management Studio",
+            "hex": "FEB900",
+            "source": "https://aleson-itc.com/wp-content/uploads/2020/03/server-m-studio.svg",
```

Comment:
I'm not super happy with this as a source.
Given it's not directly linked to Microsoft, I'd be hesitant using this. Have checked Microsoft's architecture diagram SVGs and they don't appear to be available in those packs. Are you aware of an official Microsoft source we can use?

Reply:
I'm not sure why it's not part of their architecture diagrams. Because this icon is their website and on the application once it's installed on your local PC.

For example, as you can see on [this website](https://learn.microsoft.com/en-us/sql/ssms/sql-server-management-studio-ssms?view=sql-server-ver16), it has the below image.

![](https://learn.microsoft.com/en-us/sql/ssms/media/sql-server-management-studio-ssms/ssms.png?view=sql-server-ver16)

Or this image:

![](https://learn.microsoft.com/en-us/sql/ssms/tutorials/media/ssms-configuration/objectexplorer.png?view=sql-server-ver16)

Or this one too:

![](https://learn.microsoft.com/en-us/azure/azure-sql/database/media/connect-query-ssms/connect.png?view=azuresql-mi)

They all have the icon in the top-left corner of the image. And it's there when I'm using the app for my daily work.

Reply:
🤔 It looks to me as though there is _no_ official logo or icon here, which is backed up by @adamrusted not being able to find one in Microsoft's architecture diagram SVGs and its [Wikipedia entry](https://en.wikipedia.org/wiki/SQL_Server_Management_Studio) displaying a screenshot rather than a logo. To me it looks like the toolbar/taskbar icon in the screenshots above is just a generic icon so that there's _something_ to represent it.